dev-util/edb-debugger/edb-debugger-0.9.21.ebuild

```diff
@@ -62,7 +62,7 @@ src_configure() {
 }
 
 src_install() {
-	cd src/images/ || die
+	cd src/images/
```

Comment:
Actually, `||die` was correct here.

Reply:
Any informative error message I need to write?

Reply:
Nope. The default message will be helpful enough, and it is unlikely to happen anyway.